internal/testutils/test_utils.go

```diff
@@ -51,46 +48,50 @@ func GetSamplePubKey() *authservice.PubKey {
 
 // GetSamplePubKeyFingerprint returns a sample public key fingerprint
 func GetSamplePubKeyFingerprint() string {
-	return "SHA256:I3HxcO3FpUM6MG7+rCASuePfl92JEcdz2htV7SP0Y20="
+	return "SHA256:8Kxg2cK9x03ofiEevL0hwAoW330xdy4vjD713fLLkzs="
 }
 
 // GetSamplePrivKeyPem returns a sample private key PEM
 //
 //	Matches GetSamplePubKeyPem
 func GetSamplePrivKeyPem() string {
 	return `-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Was there a reason this key was replaced? We need to make sure that we are able to load keys generated using our old process like the one here still work with the new system.